appveyor-install-environment.ps1

```diff
@@ -0,0 +1,38 @@
+#
```

Comment:
We should probably move this to sripts/appveyor/appveyor-install-environment.ps1?